src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -47,7 +47,7 @@ public ResponseEntity<WalletDTO> createWallet(@Valid @RequestBody CreateWalletDT
     ResponseEntity<List<WalletDTO>> getWallets() {
 
         List<WalletDTO> walletsDTO = walletService.getWallets();
-        HttpHeaders headers =  new HttpHeaders();
+        HttpHeaders headers = new HttpHeaders();
 
         if (!walletsDTO.isEmpty()) {
```

Comment:
This isn't my Task.
This is Task to getWallets.
Change were in this place, because in one line were two white spaces (change  during formatted).
Should I change this condition ?

Reply:
Corrected